cmd/collectors/zapiperf/zapiperf.go

```diff
@@ -1168,7 +1168,7 @@ func (z *ZapiPerf) addCounter(counter *node.Node, name, display string, enabled
 		// ONTAP does not have a `type` for histogram. Harvest tests the `desc` field to determine
 		// if a counter is a histogram
 		isHistogram = false
-		if len(labels) > 0 && strings.Contains(description, "histogram") {
+		if len(labels) > 0 && strings.Contains(strings.ToLower(description), "histogram") {
```

Comment:
can we `toLower` once, when we set the description instead of on every iteration?